pycontrails/models/cocip/cocip.py

```diff
@@ -538,10 +538,11 @@ def _process_flight(self, met: MetDataset) -> None:
         scale_humidity = humidity_scaling is not None and "specific_humidity" not in self.source
         verbose_outputs = self.params["verbose_outputs"]
 
-        interpolate_met(met, self.source, "air_temperature", **self.interp_kwargs)
-        interpolate_met(met, self.source, "specific_humidity", **self.interp_kwargs)
-        interpolate_met(met, self.source, "eastward_wind", "u_wind", **self.interp_kwargs)
-        interpolate_met(met, self.source, "northward_wind", "v_wind", **self.interp_kwargs)
+        interp_kwargs = {"q_method": self.params["interpolation_q_method"], **self.interp_kwargs}
```

Comment:
oh interesting! You do include it in `interp_kwargs` here

Reply:
Yeah ... you can see I thought about this previously, but something wasn't quite working. Anyway, this is a little cleaner now.

91a567cf0230cca62bf9e2527107008cbb69992a